README.adoc

```diff
@@ -15,6 +15,8 @@
 :page-tags: ['MicroProfile']
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['rest-intro', 'cdi-intro', 'microprofile-config', 'circuit-breaker']
+:page-seo-title: Building fault-tolerance microservices with the MicroProfile Fault Tolerance @Fallback annotation
+:page-seo-description: A tutorial and an example on how to build resilient microservices using MicroProfile Fault Tolerance and obtain metrics for the Fault Tolerance methods with MicroProfile Metrics.
```

Comment:
Change to:
"A tutorial and example on how to use MicroProfile Fault Tolerance to build resilient microservices and obtain metrics from the fault tolerance methods with MicroProfile Metrics."

Reply:
fixed